docs/rfcs/rfc-009-graphql-ui-poc.md

```diff
@@ -0,0 +1,112 @@
+# GraphQL Graph App PoC
+
+## Problem
+
+The [GraphQL Architect](https://grandstack.io/docs/graphql-architect-overview/#:~:text=GraphQL%20Architect%20is%20a%20graph,Neo4j%20Desktop%20Graph%20Apps%20Gallery) for usage in the Neo4j Desktop is due to be replaced. As an initial step a proof of concept of a successor shall be established.
+
```

Comment:
It is not completely clear where this is going to be available. Will it be a local server, webpage, desktop app...?

Maybe it is worth defining here.